package.json

```diff
@@ -41,41 +53,29 @@
     "babel-plugin-transform-flow-strip-types": "^6.18.0",
     "babel-polyfill": "^6.23.0",
     "babel-preset-es2015": "^6.6.0",
-    "chai": "^4.0.1",
+    "chai": "^4.0.2",
+    "cross-env": "^5.0.1",
     "eslint": "^3",
     "eslint-plugin-flowtype": "^2.30.0",
     "flow-bin": "^0.47.0",
     "flow-runtime": "^0.12.0",
-    "gulp": "^3.9.1",
-    "gulp-eslint": "^3.0.1",
-    "gulp-flowtype": "^1.0.0",
-    "gulp-karma": "0.0.5",
-    "gulp-webserver": "^0.9.1",
+    "flow-typed": "^2.1.2",
     "istanbul-instrumenter-loader": "^2.0.0",
     "karma": "^1.7.0",
+    "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^2.1.1",
-    "karma-commonjs": "^1.0.0",
-    "karma-coverage": "^1.0.0",
+    "karma-coverage-istanbul-reporter": "^1.3.0",
     "karma-firefox-launcher": "^1.0.0",
-    "karma-ie-launcher": "^1.0.0",
     "karma-mocha": "^1.0.1",
     "karma-phantomjs-launcher": "^1.0.0",
-    "karma-safari-launcher": "^1.0.0",
```

Comment:
Any reason to remove all of these launchers? It's useful to be able to run sanity checks against other browsers.